ebean-core/src/main/java/io/ebeaninternal/server/query/CQueryBuilder.java

```diff
@@ -760,6 +760,10 @@ boolean isPlatformDistinctOn() {
     return dbPlatform.isPlatform(Platform.POSTGRES);
   }
 
+  boolean isPlatformDistinctNoLobs() {
+    return dbPlatform.isPlatform(Platform.DB2); // CHECKME: Also oracle?
```

Comment:
I did not yet test that in oracle. That may be also delegated to DatabasePlatform
Maybe returning a enum: 
```java
enum DistinctMode { NORMAL, DISTINCT_ON, UNSELECT_LOBS}
```

Reply:
Hmmm, tests all passing with Oracle I believe (last local run, odd problems running oracle test via github workflow)